### PR TITLE
Implement card cv view-search component

### PR DESCRIPTION
**Pull request by valmdz**
# Description

Implement the component card CV view search. If the user has not added a picture, it will display an image.
Fixes issue https://github.com/HackYourFuture-CPH/rate-my-cv/issues/38

# How to test
Please provide a short summary how your changes can be tested?

` npm run storybook`
go to http://localhost:3007

# Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

File: src/client/components/CardCvViewSearchComponent/CardCvViewSearchComponent.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import unknownUser from '../../assets/images/portraits/unknown-black.jpg';
+import './CardCvViewSearchComponent.styles.css';
+import { Button } from '../Button/Button';
```
Comment (benna100) [inline src/client/components/CardCvViewSearchComponent/CardCvViewSearchComponent.js:5]: There is an error here. that should be fixed!

**Review by benna100 — CHANGES_REQUESTED** [review]

File: src/client/components/CardCvViewSearchComponent/CardCvViewSearchComponent.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+export const CardCvViewSearchComponent = ({
+  profileImageUrl,
+  fullName,
+  position,
+  cvTitle,
+  fileUrl,
+  createdDate,
+}) => {
+  const parsedDate = (date) =>
+    new Date(date).toLocaleDateString('en-GB', { timeZone: 'UTC' });
```
Comment (pdsorensen) [inline src/client/components/CardCvViewSearchComponent/CardCvViewSearchComponent.js:15]: You shouldn't have to provide a timezone, `new Date(date).toLocaleDateString();` should be sufficient.

File: src/client/components/CardCvViewSearchComponent/CardCvViewSearchComponent.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  );
+};
+
+CardCvViewSearchComponent.propTypes = {
+  profileImageUrl: PropTypes.string.isRequired,
+  fullName: PropTypes.string.isRequired,
+  position: PropTypes.string.isRequired,
+  cvTitle: PropTypes.string.isRequired,
+  fileUrl: PropTypes.string.isRequired,
+  createdDate: PropTypes.string.isRequired,
```
Comment (pdsorensen) [inline src/client/components/CardCvViewSearchComponent/CardCvViewSearchComponent.js:58]: I think it's better design to expect a date. That way you move the responsibility to upstream users, i.e the ones that will use your component.

**Review by pdsorensen — COMMENTED** [review]
This is looking good! There is quite a lot of css rules, maybe some of the margins can be removed? 

Cheers,

File: src/client/components/CardCvViewSearchComponent/CardCvViewSearchComponent.js (hunk 2 above)
Comment (valmdz) [reply src/client/components/CardCvViewSearchComponent/CardCvViewSearchComponent.js]: Ok, I have updated the files. Thank you for the feedback @pdsorensen

**Review by pdsorensen — APPROVED** [review]
Looks good. :)

**Pull request merged**